distributed/tests/test_asyncprocess.py

```diff
@@ -163,7 +163,8 @@ async def test_exitcode():
 
 
 def assert_exit_code(proc: AsyncProcess, expect: signal.Signals) -> None:
-    if WINDOWS:
+    # Note: can't use WINDOWS constant as it upsets mypy
+    if sys.platform == "win32":
```

Comment:
Is it worth instead writing `# type: ignore` ?

Reply:
I'd need to explain why there's a type: ignore anyway, and this way mypy actually works.

Reply:
Maybe we could do something like this ( https://github.com/dask/distributed/pull/7091 )?